### PR TITLE
Allow uploading the latest image under a common name

### PR DESCRIPTION
**Pull request by kashifest**
This PR makes sure that the latest image created in Openstack is pushed under a common name. This will make sure that the image being used in Jenkins workers remain active all the time. When replacing an old image with the latest, this commit also renames the old image based on its original name which is build date suffixed and this will be added in the images as an image_name property.

### CODE REVIEW

File: jenkins/image_building/upload-ci-image.sh [hunk 1]
```diff
@@ -68,8 +80,18 @@ upload_ci_image_xerces() {
   export OS_USER_DOMAIN_NAME="xerces"
   export OS_IDENTITY_API_VERSION=3
 
+ # Check if the common image already exists
+  if openstack image show "${COMMON_IMAGE_NAME}" &>/dev/null; then
```
Comment (Rozzii) [inline jenkins/image_building/upload-ci-image.sh:84]: Have you checked, does this return non zero if it can't find any image?

Comment (kashifest) [reply jenkins/image_building/upload-ci-image.sh:84]: Yes

**Review by tuminoid — COMMENTED** [review]
/approve

This is great, puts stop to Jenkins worker breakages. Thanks!

**Pull request merged**